scalyr_agent/builtin_monitors/journald_monitor.py

```diff
@@ -198,7 +198,7 @@ class JournaldMonitor(ScalyrMonitor):
 
     def _initialize(self):
         self._journal_path = self._config.get( 'journal_path' )
-        if not os.path.exists( self._journal_path ):
+        if self._journal_path and not os.path.exists( self._journal_path ):
```

Comment:
Hmm.. there's currently no way that I can see of specifying `None` for the journal path via the configuration file (since it defaults to `/var/log/journald`).  It can be empty string (which would at least let your condition pass) but I think we need to pass in None to the library to be safe.  How about we add a line above this that says:

    if len(self._journal_path) == 0:
        self._journal_path = None

Then just change the documentation for the `journal_path` configuration option to indicate empty string will use the system library.

The other choice, of course, is to change the default of `journal_path` to be None.  However, I'm not a big fan of changing default values in case customers depend on the current defaults.

Reply:
Your suggestion sounds good. Let me change the code.